README.md

```diff
@@ -59,8 +59,6 @@ Tests
 
 Papa Parse is under test. Download this repository, run `npm install`, then `npm test` to run the tests.
 
-The continuous integration server runs these tests using Node 0.12. You should ensure your tests pass locally using that version of Node.
```

Comment:
This hasn't been accurate since https://github.com/mholt/PapaParse/commit/be7e72cfd4baf3c25d8631b5823e1b4ae88364be